Replicator/tests/data/ssl_walrus_config_3-0.json

```diff
@@ -8,8 +8,8 @@
 	"adminInterface": ":4995",
 	"metricsInterface": ":4996",
 	"admin_interface_authentication": false,
-	"SSLCert": "sg_cert.pem",
-	"SSLKey":  "sg_key.pem",
+	"SSLCert": "cert/sg_cert.pem",
```

Comment:
If the sg_cert.* at the data folder are not used anymore, please delete.

Reply:
Sorry, yeah, I thought I did that